### DISCUSSION TITLE
faro-demo not building up

### DISCUSSION THREAD

**Discussion by decid** [grafana/faro-web-sdk#272] [Q&A]
Hi, just to let you know, it looks like faro-demo is unable to be built up using docker compose, below are the logs:
(https://paste.ofcode.org/LVjMkuvWfw4Dd23NjAPCVB)

**Comment by eskirk** [comment]
> Hi, just to let you know, it looks like faro-demo is unable to be built up using docker compose, below are the logs: (https://paste.ofcode.org/LVjMkuvWfw4Dd23NjAPCVB)

hi @decid thank you for the feedback - just merged #274, hopefully that fixes your issue. if not please let me know - thanks!
